.gitignore

```diff
@@ -3,3 +3,4 @@ tmp/
 /npm-debug.log
 .directory
 .kateproject
+.idea/
```

Comment:
We should remove `.directory` and `.kateproject` as well, unless they're actually generated by this project.